frontend/app/components/task-selection/component.js

```diff
@@ -116,6 +136,13 @@ export default class TaskSelectionComponent extends Component {
     } else {
       this.tracking.fetchCustomers.perform();
     }
+
+    if (this.customer?.id) {
+      this.customer.project = this.customerProjectsData.value ?? [];
+    }
+    if (this.project?.id) {
+      this.project.tasks = this.projectTasksData.value ?? [];
+    }
```

Comment:
@MitanOmar Is this copy :spaghetti:  from the `tracking` service? 
This duplicates requests and does not respect what is done in the code above. I think an include as proposed by Arthur or a fix in the `onCustomerChange` method is necessary. Why is this error surfacing and under what circumstances (please don't quote the issue)?